src/DB/exerciseData.json

```diff
@@ -12,6 +12,16 @@
     "gh-name": "druvkotwani",
     "image": "https://www.inspireusafoundation.org/wp-content/uploads/2023/01/inner-bicep-curl.gif"
   },
+  {
+    "exercise": "Heel Touches",
+    "instructions": ["Laying flat on your back, place your hands behind your head. Bend your knees and bring them up into 
```

Comment:
Should be in a single line for each point and it requires a comma to add more. You can also take references from this file

Reply:
sorry i forget it was a new mistake as a newbie